.github/workflows/url-validation.yml

```
@@ -28,4 +25,18 @@ jobs:
               exit 1
             fi
             echo "OK:    $url"
+          done
+          
+          # Check local files exist instead of GitHub Pages URLs
+          files=(
+            "docs/schemas/xdebug-debug.json"
+            "docs/debug-guidelines.md"
+          )
+          
```

YAMLlint (1.37.1)

[error] trailing spaces

(trailing-spaces)